app/ante.go

```diff
@@ -62,26 +68,22 @@ func GetAnteDecorators(options HandlerOptions) []sdk.AnteDecorator {
 	return anteDecorators
 }
 
-// NewAnteHandler returns an AnteHandler that checks and increments sequence
-// numbers, checks signatures & account numbers, and deducts fees from the first
-// signer.
-func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
+func (o HandlerOptions) validate() error {
```

Comment:
missing validation for used options:
`options.TxCounterStoreKey`
`options.SigGasConsumer`